src/main/resources/META-INF/plugin.xml

```diff
@@ -1,6 +1,6 @@
 <idea-plugin>
-	<id>com.liferay.ide.idea.plugin</id>
-	<name>Liferay Idea Plugin</name>
+	<id>com.liferay.ide.intellij.plugin</id>
```

Comment:
Hey @jtydhr88  is this going to be a problem now that we have already uploaded the Liferay Intellij plugin ?